tests/manage/z_cluster/test_kill_ceph_mon_deamon.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1, bugzilla
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.helpers.helpers import verify_cli_cmd_output
+from ocs_ci.ocs.resources.pod import (
+    get_pod_node,
+    get_mon_pods,
+    get_ceph_tools_pod,
+    get_mgr_pods,
+    get_osd_pods,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("1904917")
+class TestKillCephMonDaemon(ManageTest):
+    """
+    Verify coredump getting generated for ceph daemon crash
+
+    """
+
+    @pytest.fixture(scope="function")
+    def teardown(self, request):
+        """
+        Archive a crash report so that it is no longer considered for the
+        RECENT_CRASH health check and does not appear in the
+        crash ls-new output
+
+        """
+
+        def finalizer():
+            tool_pod = get_ceph_tools_pod()
+            tool_pod.exec_ceph_cmd(ceph_cmd="ceph crash archive-all", format=None)
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.parametrize(
+        argnames=["daemon_type"],
+        argvalues=[
+            pytest.param(*["mon"], marks=pytest.mark.polarion_id("OCS-2491")),
```

Comment:
Please create separate test cases for each daemon type. having a same polarion ID overwrites the execution status for each run